ingestion-beam/src/main/java/com/mozilla/telemetry/Decoder.java

```diff
@@ -32,48 +30,39 @@
    * @param args command line arguments
    */
   public static void main(String[] args) {
-    // register options class so that `--help=DecoderOptions` works
+    // Register options class so that `--help=DecoderOptions` works.
     PipelineOptionsFactory.register(DecoderOptions.class);
 
     final DecoderOptions.Parsed options = DecoderOptions.parseDecoderOptions(
         PipelineOptionsFactory.fromArgs(args).withValidation().as(DecoderOptions.class));
     final Pipeline pipeline = Pipeline.create(options);
     final List<PCollection<PubsubMessage>> errorCollections = new ArrayList<>();
 
-    pipeline.apply("input", options.getInputType().read(options))
-        .apply("collect input parsing errors", PTransform.compose((PCollectionTuple input) -> {
-          errorCollections.add(input.get(DecodePubsubMessages.errorTag));
-          return input.get(DecodePubsubMessages.mainTag);
-        })).apply("parseUri", new ParseUri())
-        .apply("collect parseUri errors", PTransform.compose((PCollectionTuple input) -> {
-          errorCollections.add(input.get(ParseUri.errorTag));
-          return input.get(ParseUri.mainTag);
-        })).apply("validateSchema", new ValidateSchema())
-        .apply("collect validateSchema errors", PTransform.compose((PCollectionTuple input) -> {
-          errorCollections.add(input.get(ValidateSchema.errorTag));
-          return input.get(ValidateSchema.mainTag);
-        })).apply("decompress", new GzipDecompress())
+    // Trailing comments are used below to prevent rewrapping by google-java-format.
```

Comment:
I'm open to pushback about whether using `//` to control google-java-format behavior is a road we're willing to go down. I've been finding this Decoder class more difficult to read than it needs to be, though.

Reply:
using `//` lgtm, and i definitely agree that it's much more readable